client/src/app/+admin/moderation/video-abuse-list/video-abuse-list.component.html

```diff
@@ -40,47 +51,141 @@
+                    {videoAbuse.countReportsForReporter, plural, =1 {1 report} other {{{ videoAbuse.countReportsForReporter }} reports}}<span class="ml-1 glyphicon glyphicon-flag"></span>
+                  </a>
+                </span>
+              </div>
+              <div class="d-flex">
+                <span class="col-3 moderation-expanded-label" i18n>Reportee</span>
+                <span class="col-9 moderation-expanded-text">
+                  <div class="chip">
+                    <img
+                      class="avatar"
+                      [src]="videoAbuse.video.channel.ownerAccount?.avatar.path"
+                      (error)="switchToDefaultAvatar($event)"
+                      alt="Avatar"
+                    >
+                    <div>
+                      <span class="text-muted">{{ videoAbuse.video.channel.ownerAccount ? createByString(videoAbuse.video.channel.ownerAccount) : '' }}</span>
+                    </div>
+                  </div>
+                  <a [routerLink]="[ '/admin/moderation/video-abuses/list' ]" [queryParams]="{ 'search': videoAbuse.video.channel.ownerAccount.displayName }" class="ml-auto text-muted video-details-links" *ngIf="!videoAbuse.video.deleted" i18n>
+                    {videoAbuse.countReportsForReportee, plural, =1 {1 report} other {{{ videoAbuse.countReportsForReportee }} reports}}<span class="ml-1 glyphicon glyphicon-flag"></span>
+                  </a>
+                </span>
+              </div>
+              <div class="d-flex">
+                <span class="col-3 moderation-expanded-label" i18n>Updated</span>
+                <time class="col-9 moderation-expanded-text video-details-date-updated">{{ videoAbuse.updatedAt | date: 'medium' }}</time>
+              </div>
+
+              <!-- report text -->
+              <div class="mt-3 d-flex">
+                <span class="col-3 moderation-expanded-label" i18n>Report #{{ videoAbuse.id }}</span>
                 <span class="col-9 moderation-expanded-text" [innerHTML]="videoAbuse.reasonHtml"></span>
               </div>
               <div class="mt-3 d-flex" *ngIf="videoAbuse.moderationComment">
-                <span class="col-3 moderation-expanded-label" i18n>Note:</span>
+                <span class="col-3 moderation-expanded-label" i18n>Note</span>
                 <span class="col-9 moderation-expanded-text" [innerHTML]="videoAbuse.moderationCommentHtml"></span>
               </div>
             </div>
 
             <div class="col-4">
-              <div class="screenratio" [innerHTML]="videoAbuse.embedHtml"></div>
+              <div class="screenratio">
+                <div *ngIf="videoAbuse.video.deleted || videoAbuse.video.blacklisted">
+                  <span i18n>The video was {{ videoAbuse.video.deleted ? 'deleted' : 'blacklisted' }}</span>
```

Comment:
For translators, I think it would be clearer with

````
<span i18n *ngIf="videoAbuse.video.deleted">This video was deleted</span>
<span i18n *ngIf="!videoAbuse.video.deleted">This video was blacklistedspan>
````